### ISSUE TITLE
IE open science manifesto

### ISSUE THREAD

**Issue by la-sch** [IndEcol/OpenScience#7]
To me, it seems that the different working groups – data inventory, ontology, community database – are sequential tasks building on each other rather than parallel endeavours. The data inventory could compile existing data sources and their associated metadata, which facilitates data selection. For developing an ontology or alternative data model, one could first look at the existing database or ontology schemas of the inventoried data in step 1, and harmonize these instead of starting from scratch. This would make it easier to adapt existing data to the new harmonized data model, and increase its acceptance / popularity. Besides reproducibility and facilitation of meta-analysis, the data model could also aim to facilitate model / method integration, like coupling LCA and IO. Finally, instead of a community database, an information system could be developed that allows to query and extract data from the distributed, autonomous databases inventoried in step 1 thanks to the data harmonization in step 2.

**Comment by stefanpauliuk** [comment]
I agree that some review work is needed for us to better understand the existing approaches, esp. in the LCA community. Rupert started to work on a roadmap (IE open science manifesto) and it would be great if you could post your ideas there!

I think that a IE data information system should precede a community database, just because it’s much easier to set up and maintain.

Even simpler, I am thinking of starting an IE data inventory, which is just a searchable open spreadsheet where we all can add and tag our (partly) published data.
There will be a lot of data gathering on material cycles as part of the RECC-project led by Edgar, and I want to use that as an opportunity to start a systematic stocktaking process of IE data. My goal is to have something small but presentable ready for the GRC in May.

My first naïve attempt is here:
[IE data inventory template](https://docs.google.com/spreadsheets/d/1yupwhtfUiBnW5DcAzOTek03gxzH22hDzn_djzwocDGA/edit?usp=sharing )
Would like to hear your feedback!

**Comment by la-sch** [comment]
Thanks, Stefan. I made some changes to the manifesto.

Regarding the template, I suggest to split the time and region scopes both into coverage and resolution. In addition, for some columns I would add dropdown menus to avoid different spellings, and you could even think about auto-incrementing the ID column to ensure IDs are unique.

**Comment by nheeren** [comment]
@la-sch Whatever way this Wiki / group develops, it will live exclusively from its contributors. So the more important question will be who will contribute to what work stream or topic. Although, I fully agree that this needs streamlining, people (we) will not contribute if they don't see synergies to their ongoing work or their current research interests. That tells me that your point is one that ought to be discussed in the group. Therefore, I took the liberty to add the tag `AgendaPoint` to your issue.

**Comment by ricklupton** [comment]
@stefanpauliuk a few things that might be missing:
- you have flows but not stocks -- or is that included in "processes"?
- what do you have in mind for Broad group?
- the license of the data should probably be logged
- the spreadsheet should have a license to make it clear how people can use it -- perhaps [CC BY](https://creativecommons.org/licenses/by/4.0/)

Are you intending to get it started by adding some things yourself? I think that would make it easier to add to if there were some examples already.

**Comment by stefanpauliuk** [comment]
Thanks Laura and Rick four your suggestions and comments!
I will revise the template accordingly, add a license, and put in some examples. Not sure I will manage before Easter, though.
Will keep you posted and then we can decide how to proceed!

**Comment by nheeren** [comment]
In case anyone missed it. We are currently working on the manifesto and will finalise it this week. This will close this issue.

https://docs.google.com/document/d/1_UomyoSY8tM6pszJ8HHRal9JixTc2ITMUUw2luwWuyk/edit?usp=sharing
